File: docs/sdk-documentation/getting-started/import-and-initialize/java/import.java
import com.absmartly.sdk.*;

public class Example {
    static public void main(String[] args) {

        final ClientConfig clientConfig = ClientConfig.create()
            .setEndpoint("https://your-company.absmartly.io/v1")
            .setAPIKey("YOUR-API-KEY")
            .setApplication("website") // created in the ABSmartly web console
            .setEnvironment("development");  // created in the ABSmartly web console

        final Client absmartlyClient = Client.create(clientConfig);

        final ABSmartlyConfig sdkConfig = ABSmartlyConfig.create()
            .setClient(absmartlyClient);


        final ABSmartly sdk = ABSmartly.create(sdkConfig);
        // ...
    }
}

File: docs/APIs-and-SDKs/SDK-Documentation/getting-started/import-and-initialize/kotlin/import.kt
import com.absmartly.sdk.*

val clientConfig = ClientConfig.create()
    .setEndpoint("https://your-company.absmartly.io/v1")
    .setAPIKey(System.getenv("ABSMARTLY_APIKEY"))
    .setApplication(System.getenv("ABSMARTLY_APPLICATION"))
    .setEnvironment(System.getenv("ABSMARTLY_ENVIRONMENT"))

val sdkConfig = ABSmartlyConfig.create()
    .setClient(Client.create(clientConfig))

val sdk = ABsmartly.create(sdkConfig)
